org.kde.kget.json

```diff
@@ -0,0 +1,140 @@
+{
+  "id" : "org.kde.kget",
+  "branch" : "master",
+  "runtime" : "org.kde.Platform",
+  "runtime-version" : "5.15-21.08",
+  "runtime-commit" : "e8e3a76d0c26375c55740919779a70e76843cca2305ec01bc3c5d3cbeae2c9b6",
+  "sdk" : "org.kde.Sdk",
+  "sdk-commit" : "5a0b113ebd0a557ee77d73107176ff5f9489da8db422a41f03e2159d43b078a8",
+  "command" : "kget",
+  "modules" : [
+    {
+      "name" : "boost",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url" : "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2",
+          "sha256" : "f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41",
+          "type" : "archive"
+        }
+      ],
+      "build-commands" : [
+        "./bootstrap.sh --prefix=/app --with-libraries=system",
+        "./b2 -j $FLATPAK_BUILDER_N_JOBS install"
+      ]
+    },
+    {
+      "name" : "gpgme",
```

Comment:
gpgme is in the runtime.

Reply:
OK, how can i know this? So will be able to keep in mind for next time.

Reply:
is this the logic that those which doesn't have buildsystems are in runtime

Reply:
if yes then confirm that these shouldn't be included.

Reply:
https://github.com/flathub/org.kde.kontact/pull/55#issuecomment-1000528230 I need to check this again but the FDO Runtime provided gpgme is missing Qt support.